src/v/config/configuration.cc

```diff
@@ -459,6 +459,27 @@ configuration::configuration()
       "the majority.",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       true)
+  , raft_max_inflight_follower_append_entries_requests_per_shard(
+      *this,
+      "raft_max_inflight_follower_append_entries_requests_per_shard",
+      "The maximum number of append entry requests that may be sent from raft "
+      "groups on a Seastar shard to the current node and are awaiting a reply. "
+      "This property replaces "
+      "raft_max_concurrent_append_requests_per_follower.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      1024,
+      {.min = 1, .max = 50000})
+  , raft_max_buffered_follower_append_entries_bytes_per_shard(
+      *this,
+      "raft_max_buffered_follower_append_entries_bytes_per_shard",
+      "The total size of append entry requests that may be cached per shard, "
```

Comment:
Just so I'm clear, this limits exactly the same thing as the property above, except the one above counts in "requests" and this one counts in "bytes", and whichever limit is hit forms the effective in-flight limit, right?

If that's the case, I find it confusing that they have quite different descriptions, e.g., this one talking about caching but the above not mentioning caching etc. Perhaps they could have the same description varying only in the "requests/bytes" part and also referring to the the other one?

Reply:
To answer my own question, they are different. "requests" is max requests outstanding to other shard, while "bytes" is max bytes in _unsent_ requests which can be additionally be buffered w/o blocking after we hit the first limit.

@mmaslankaprv thoughts on why it makes sense to measure the first in request and the second in bytes?

Reply:
The reasoning behind that was that the size is important for the memory pressure, while the number of request being a follower queue depth. Therefore i introduced the separation to be able to control the size of the memory buffer and queue depth.

Reply:
@mmaslankaprv - thanks, but is my other comment above correct: that they also measure different parts of the queue? I.e., "bytes" is for unsent rpcs and "requests" is for sent-but-not-responded rpcs?

Reply:
exactly the comment is correct